### PR TITLE
Hook up sideshow to Google's OpenID Connect endpoint instead of OpenID endpoint.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
* Remove the `openidTool`

### CODE REVIEW

File: bin/persona-gmail-bridge.js [hunk 1]
```diff
@@ -280,6 +267,25 @@ app.get('/authenticate/forward', validate({ email: 'gmail' }),
 
 app.get('/authenticate/verify', function (req, res) {
   // Check input precondition:
+  // The bridge should return the expected state token.
+  if (req.query.state !== req.session.state) {
```
Comment (shane-tomlinson) [inline bin/persona-gmail-bridge.js]: Also check to make sure one or the other is not falsey. If both are `undefined`, doesn't really help.

File: bin/persona-gmail-bridge.js [hunk 2]
```diff
@@ -280,6 +267,25 @@ app.get('/authenticate/forward', validate({ email: 'gmail' }),
...
+    logger.error('Invalid or missing state');
+    statsd.increment('authentication.oauth.failure.no_state');
+    return res.status(400).render('error',
+      { title: req.gettext('Error'), errorInfo: 'Invalid or missing state.' });
+  }
+  // a state token is only good for one request
+  delete req.session.state;
+
+  var code = req.query.code;
```
Comment (shane-tomlinson) [inline bin/persona-gmail-bridge.js]: Comment this.

File: test/lib/mockid.js [hunk 3]
```diff
@@ -2,25 +2,54 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const assert = require('assert');
```
Comment (shane-tomlinson) [inline test/lib/mockid.js]: Update the tests!

**Comment by callahad** [comment]
Closing this pull request and switching development to the [openid-connect branch in this repo](https://github.com/mozilla/persona-gmail-bridge/tree/openid-connect), so that we can collaborate on a few lingering issue. Especially #127, #128, and #129.

**Pull request closed without merging**